src/pybind/mgr/balancer/module.py

```diff
@@ -221,6 +221,22 @@ class Module(MgrModule):
             'long_desc': 'This is a time of day in the format HHMM.',
             'runtime': True,
         },
+        {
+            'name': 'begin_weekday',
+            'type': 'uint',
+            'default': 0,
+            'desc': 'Restrict automatic balancing to this day of the week or later',
+            'long_desc': '0 or 7 = Sunday, 1 = Monday, etc.',
```

Comment:
Hi @xiexingguo, a small bug here: in Python when `time.localtime().tm_wday == 0` it's actually Monday.

Reply:
See https://github.com/ceph/ceph/pull/26505/commits/b9da0925490d4e9e964249c54e7775d9c76b1719

Reply:
Ah... totally missed it. Thanks and sorry for the noise!